Rosalina hitbox edits

FSFrozen opened the pull request:
Bair: Late hit terminates 5 frames sooner (ends on frame 16 from 21)

Uair: Motion rate added to end of animation (+4 endlag)

Utilt: Late hit terminates 2 frames earlier (ends on frame 15 from 17)

FSFrozen committed:
Rosalina hitbox edits

Bair: Late hit terminates 5 frames sooner (ends on frame 16 from 21)

Uair: Motion rate added to end of animation (+4 endlag)

Utilt: Late hit terminates 2 frames earlier (ends on frame 15 from 17)

FSFrozen committed:
Further rosa adjustments

Full changelog

Rosa:
Utilt: Late hit active frames reduced by 2 (terminates on 15 from 17)

Bair: Angle 30->40 on all hits, strong hit KBG 83->86, late hit active frames reduced by 5 (terminates on 16 from 21)

Uair: +4 endlag (FAF 49->53)

Dash attack hit 2: Z-position on outermost hitbox reigned in by 3 units, active frames reduced by 2 (Terminates on 21 from 23)

Luma:
HP 60->50

Detethered only: KB threshold to launch reduced
[free_damage_for_reaction 30->70, free_damage_air_brake 0.07->0.15]

Spawn rate for different colors normalized (less chance of yellow, higher chance of black/white)

Z-hime commented:
Very good step in the right direction.
Luma kb thresholds and hp differences make it apparent that Rosalina now has to protect Luma much more than before. Matchups seem to feel a bit less one sided in the fact that Luma is able to be disposed of a bit easier now.

Uair changes are great to prevent several staircase combos against characters that may not be fast falling enough to deserve them. Overall felt much more enjoyable to fight as you can focus your energy more on Rosalina rather than trying to get through Luma walls.

LGTM!

hdr-balance-backroom commented on fighters/rosetta/src/acmd/aerials.rs:
```diff
@@ -126,6 +126,7 @@ unsafe fn rosetta_attack_air_hi_game(fighter: &mut L2CAgentBase) {
     }
     frame(lua_state, 45.0);
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 4.0);
```
This motion rate should be set earlier in the move with a smaller multiplier to avoid strange animation inconsistencies

hdr-balance-backroom commented:
Please include a full changelog in the main body once complete

FSFrozen committed:
small uair fix

motion rate moved to earlier in the animation to look less awkward towards the end

SkewedAskew reviewed (dismissed):
Dash attack still feels too disjointed. As an idea, try changing the outermost hitbox to a set of two capsules like so:

![image](https://user-images.githubusercontent.com/55216313/213402294-ed8c60a4-86d2-4401-afd1-59702eec3c9b.png)

Otherwise lgtm

SkewedAskew committed:
Merge branch 'dev' into pr/1298

SkewedAskew committed:
Dash attack

SkewedAskew reviewed (approved):
Lgtm
